css/selectors/focus-visible.json

```diff
@@ -11,7 +11,7 @@
               "flags": [
                 {
                   "name": "#enable-experimental-web-platform-features",
-                  "type": "runtime_flag",
+                  "type": "preference",
```

Comment:
I agree with this change, but I don't think this change belongs in this PR.  Can we separate this into another PR?